cas-server-documentation/installation/LDAP-Authentication.md

```diff
@@ -10,12 +10,12 @@ LDAP integration is enabled by including the following dependency in the Maven W
 <dependency>
      <groupId>org.jasig.cas</groupId>
      <artifactId>cas-server-support-ldap</artifactId>
-     <version>${cas.version}</version>
+     <version>${project.version}</version>
```

Comment:
Just make the changes and push new commits to your fork's `4.1.x` branch. The updated commits will show up in this PR automatically.